src/current/v23.1/backup-architecture.md

```diff
@@ -57,7 +57,7 @@ Once one of the nodes has claimed the job from the system jobs table, it will ta
 - Determine the specific subdirectory for this backup, including if it should be incremental from any discovered existing directories.
```

Comment:
I realize in v23.1 of this docs, I have made the updates without keeping the prior content in place. From checking the tags, it looks like the code base change only applies to v23.1.14+... Should I keep the existing content in place, and keep the updated replica workflow in its own "new-in" tag on this page? Or, since v23.1 is going out of support, go ahead and update as per this commit?

Reply:
we didn't change any behavior by default in 23.1.x backports -- the backport there is default off and a non-public cluster setting so I don't think we want to update any 23.1 docs.